inline_scan.sh

```diff
@@ -485,24 +478,24 @@ print_scan_result_summary_message() {
     if [[ ! "${V_flag-""}"  && ! "${R_flag-""}" ]]; then
         if [[ ! "${status}" = "pass" ]]; then
             echo "Result Details: "
-            curl -s -k --header "Content-Type: application/json" -H "Authorization: Bearer ${SYSDIG_API_TOKEN}" "${SYSDIG_ANCHORE_URL}/images/by_id/${SYSDIG_IMAGE_ID}/check?tag=$FULLTAG&detail=true"
+            curl -s -k --header "Content-Type: application/json" -H "Authorization: Bearer ${SYSDIG_API_TOKEN}" "${SYSDIG_ANCHORE_URL}/images/by_id/${SYSDIG_IMAGE_ID}/check?tag=${REPO_TAG}&detail=true"
         fi
-        ENCODED_TAG=$(urlencode ${FULLTAG})
-
-        if [[ -z "${clean_flag:-}" || "${status}" = "pass" ]]; then
-            if [[ "${o_flag:-}" ]]; then
-                    echo "View the full result @ ${SYSDIG_BASE_SCANNING_URL}/secure/#/scanning/scan-results/${ENCODED_TAG}/${SYSDIG_IMAGE_DIGEST}/summaries"
-            else
-                echo "View the full result @ ${SYSDIG_BASE_SCANNING_URL}/#/scanning/scan-results/${ENCODED_TAG}/${SYSDIG_IMAGE_DIGEST}/summaries"
-            fi
+    fi
+
+    if [[ -z "${clean_flag:-}" ]]; then
```

Comment:
Here I've just fixed the "full result url" not being showed most of the time.

Now it appears every time but when passing -C flag.